eng/Version.Details.xml

```diff
@@ -27,5 +27,53 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>731d793be2d0a66bafc96b1a79dc96b4d1f0301b</Sha>
     </Dependency>
+    <!--
+      Maui Rollback Version mappings, default means generated from sdk version. Allows for manual override of version similar to https://github.com/dotnet/maui/blob/df8cfcf635a590955a8cc3d6cf7ae6280449dd3f/eng/Versions.props#L138-L146, where the logic comes from:
+      Mapping_Microsoft.Maui.Controls:default
+      Mapping_Microsoft.NETCore.App.Ref:default
+      Mapping_Microsoft.NET.Workload.Emscripten.Current:default
+      Mapping_Microsoft.Android.Sdk:default
+      Mapping_Microsoft.MacCatalyst.Sdk:9.0.100-preview.6
+      Mapping_Microsoft.macOS.Sdk:9.0.100-preview.6
+      Mapping_Microsoft.iOS.Sdk:9.0.100-preview.6
+      Mapping_Microsoft.tvOS.Sdk:9.0.100-preview.6
```

Comment:
How is this information generated? Is this done by Maestro or is this something we will need to manually update?

Reply:
Yes, this is something we will have to manually update when Maui has to hardcode the data themselves. These values will have to match whatever is being generated in place of https://github.com/dotnet/maui/blob/net9.0/eng/Versions.props#L141-L155. Though this should be rare throughout the release cycle. The comment for these mappings does link to a specific snapshot of the values which can be used to figure out what the current values should be.